src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  let currentStateCopy = { ...state };
+
+  actions.forEach((action) => {
+    switch (action.type) {
```

Comment:
Use destructuring for `action` object
```suggestion
  actions.forEach((action) => {
    const { type, ....... } = action;
    
    switch (type) {
```